src/components/authorization/roles/staff-member.role.ts

```diff
@@ -274,7 +274,7 @@ export const StaffMember = new DbRole({
       canList: true,
       properties: [
         { propertyName: 'organization', permission: { read, }, },
-        { propertyName: 'pointOfContact', permission: { read, }, },
+        { propertyName: 'pointOfContact', permission: { }, },
         { propertyName: 'types', permission: { read, }, },
         { propertyName: 'financialReportingTypes', permission: { read, }, },
         { propertyName: 'pmcEntityCode', permission: { read, }, },
```

Comment:
Again `pointOfContact` looks good; you can do the same with `pmcEntityCode`.  And also take `organization` down to Low Sensitivity.